build.cmd

```diff
@@ -242,12 +242,10 @@ set __RunArgs=-BuildOS=%__BuildOS% -BuildType=%__BuildType% -BuildArch=%__BuildA
 
 if %__EnforcePgo%==1 (
     if %__BuildArchArm%==1 (
-        echo Error: enforcepgo cannot be used with arm architecture
-        goto Usage
+        echo WARNING: enforcepgo does nothing with arm architecture
```

Comment:
Does this actually emit a warning, or just the text "WARNING"? I can see getting warned maybe being useful for a dev, but in official builds this path is normal and we recently got down to 0 warnings.